webapp/channels/src/components/common/hooks/useOpenPricingModal.ts

```diff
@@ -31,14 +40,17 @@ export default function useOpenPricingModal() {
         trackEvent(category, 'click_open_pricing_modal', {
             callerInfo: telemetryProps?.trackingLocation,
         });
-        dispatch(openModal({
-            modalId: ModalIdentifiers.PRICING_MODAL,
-            dialogType: PricingModal,
-            dialogProps: {
-                callerCTA: telemetryProps?.trackingLocation,
-            },
-        }));
-    }, [dispatch, isCloud]);
-
-    return openPricingModal;
+
+        if (canAccessExternalPricing) {
+            // Redirect to external pricing page
+            window.open('https://mattermost.com/pricing', '_blank', 'noopener,noreferrer');
```

Comment:
Do we need to include the query params we add with the `ExternalLink` component?

Reply:
Updated to use the useExternalLink hook.